tests/foreman/cli/test_repository.py

```diff
@@ -159,6 +168,45 @@ def test_positive_create_with_puppet_repo(self):
 
     @run_only_on('sat')
     @tier1
+    def test_positive_create_with_auth_yum_repo(self):
+        """Create YUM repository with basic HTTP authentication
+
+        @Feature: Repository
+
+        @Assert: YUM repository is created
+        """
+        for creds in valid_http_credentials(url_encoded=True):
+            url = FAKE_5_YUM_REPO
```

Comment:
Since this value don't change on every iteration is better to move outside the loop.